tests/src/main/java/com/hazelcast/simulator/tests/map/IntByteMapTest.java

```diff
@@ -58,6 +59,7 @@
 
     private IMap<Integer, Object> map;
     private int[] keys;
+    private byte[][] values;
```

Comment:
I missed there are independent `keyCount` and `valueCount` properties. 
This comment can be considered RESOLVED.